CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.24)
+cmake_minimum_required(VERSION 3.30)
 
```

Comment:
You didn't merge in the switch back to 3.24 here (and in `external/CMakeLists.txt`.  That's why the debug builder is failing.